error-messages/user.messages.js

```diff
@@ -12,6 +12,7 @@ const AUTHENTICATION_TOKEN_NOT_VALID = 'AUTHENTICATION_TOKEN_NOT_VALID';
 const RESET_PASSWORD_TOKEN_NOT_VALID = 'RESET_PASSWORD_TOKEN_NOT_VALID';
 const USER_EMAIL_ALREADY_CONFIRMED = 'USER_EMAIL_ALREADY_CONFIRMED';
 const INVALID_ADMIN_INVITATIONAL_TOKEN = 'INVALID_ADMIN_INVITATIONAL_TOKEN';
+const SUPERADMIN_IS_IMMUTABLE = 'SUPERADMIN_IS_IMMUTABLE';
```

Comment:
Please, rename it to `SUPER_ADMIN_IS_IMMUTABLE`

Reply:
fixed